Mobile/starter_project_dark_knights/lib/features/signup/presentation/widgets/navigation.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/material.dart';
+import '../../../../core/utils/colors.dart';
+import '../screen/pages/signup.dart';
+
+class TopNavigate extends StatelessWidget {
```

Comment:
rename to NavigationBar or TopNavigationBar or SignUpNavBar

Reply:
Done